src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Referral.kt

```diff
@@ -67,6 +69,13 @@ class Referral(
 
   @OneToOne(mappedBy = "referral", optional = true) @Fetch(JOIN) var actionPlan: ActionPlan? = null,
 
+  @ManyToMany(fetch = FetchType.LAZY)
```

Comment:
how come this is manytomany instead of onetomany?

Reply:
A referral can have multiple service categories selected by the user, and a service category can be linked to multiple referrals.

Reply:
also, do we need to store an entity relationship here? i think we can just store the ids? might make the querying more efficient

Reply:
We decided on using entity relationship over UUID on slack